modules/installation-alibaba-permissions.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-account.adoc
+
+[id="installation-aws-permissions_{context}"]
+= Required Alibaba permissions for the RAM user
+
+.Question
+
+Is the following list of Alibaba permissions complete or are additional permissions required?
+
+* AliyunOSSFullAccess
+* AliyunECSFullAccess
+* AliyunVPCFullAccess
+* AliyunSLBFullAccess
+* AliyunEIPFullAccess
+* AliyunPubDNSFullAccess
+* AliyunNATGatewayFullAccess
+* AliyunDNSFullAccess
+* AliyunImageSearchFullAccess
+* AliyunDomainFullAccess
+* AliyunPvtzFullAccess
+* AliyunRAMFullAccess
```

Comment:
I don't believe that we should provide that `*FullAccess` permissions on the document. The procedure to grant the correct permissions for each user created by CCO, alongside the RAM user's permissions to run the CCO is available on [CCO utility document](https://github.com/DahuK/cloud-credential-operator/blob/de9cc738e2183f483819a062ce2c5175ea274e3b/docs/ccoctl.md#prerequisite).